copyright/copyright.js

```diff
@@ -0,0 +1,106 @@
+/* * *  *  * *  *  *  *  *  *  *  *  *  *  *  *  *  *  *  *  *  *  *  *  *  * */
+/* Copyright (c) 2017 Mobify Research & Development Inc. All rights reserved. */
+/* * *  *  * *  *  *  *  *  *  *  *  *  *  *  *  *  *  *  *  *  *  *  *  *  * */
+
+const glob = require('glob')
+const fs = require('fs')
+
+const copyright = {
+    items: [],
+    missingHeaders: [],
+    copyrightText: '',
+    lintMode: false,
+    updateHeaders() {
+        const filesRead = this.items.map((item) => {
+            return new Promise((resolve) => {
+                fs.readFile(item, (err, data) => {
+                    if (!this.hasCopyrightHeader(data)) {
+                        if (this.lintMode === true) {
+                            console.log(`\x1b[33m Missing copyright headers in \x1b[36m ${item}`)
```

Comment:
It'd be good to use constants for these escape codes.  Without running the tool I have no idea what `\x1b[33m` does.  :)